src/conn.c

```diff
@@ -725,7 +729,9 @@ _makeTLSConn(natsConnection *nc)
 #endif
             if (nc->tlsName != NULL)
             {
-#if defined(NATS_USE_OPENSSL_1_1)
+// the SSL_set_hostflags function is not available in the
+// wolfSSL OpenSSL compatibility layer, so use the new function instead
+#if defined(NATS_USE_OPENSSL_1_1) && !defined(NATS_HAS_WOLFSSL)
```

Comment:
Related to the comment in the main CMakeFile.txt. If `NATS_USE_OPENSSL_1_1` does not make sense in the context of WolfSSL, then it should not have been set in the make file.